avocado/plugins/run.py

```diff
@@ -185,4 +186,13 @@ def run(self, args):
             log.error(e.message)
             sys.exit(exit_codes.AVOCADO_FAIL)
         job_instance = job.Job(args)
-        return job_instance.run()
+        job_run = job_instance.run()
+        result_dispatcher = ResultDispatcher()
+        if result_dispatcher.extensions:
+            # At this point job_instance doesn't have a single results attribute
```

Comment:
OK, so the single results attribute means that currently results (test statues) are stored per each output_plugin in `results_proxy` and the target goal is to remove the `result_proxy` and store the overall results directly in `job_instance`, right?